neo.UnitTests/TestUtils.cs

```diff
@@ -28,7 +28,11 @@ public static Transaction GetTransaction()
                 Script = new byte[1],
                 Sender = UInt160.Zero,
                 Attributes = new TransactionAttribute[0],
-                Witnesses = new Witness[0]
+                Witness = new Witness
```

Comment:
We can rename `Witness` to `Signature`? (for example)

Reply:
`Witness` is not necessarily a signature.